src/ib_module.F

```diff
@@ -1893,7 +1893,7 @@ subroutine ib_lwr_weno(stag,ix,jy,kz,c1,c2,rrw,dumz,a,kbdy,vadvorder,weps)
 
   ELSEIF( stag.eq.4 )THEN
 
-    !$acc parallel loop private(i,j,k,kval,wbar)
+    !$acc parallel loop default(present) private(i,j,k,kval,wbar)
```

Comment:
maybe add `gang vector` clause and `collapse` clause, and remove `private` clause?